pages/public_cloud/containers_orchestration/managed_kubernetes/forwarding-audit-logs-to-logs-data-platform/guide.en-gb.md

```diff
@@ -0,0 +1,216 @@
+
+The OVHcloud Managed Kubernetes Service has defined an audit policy, enabling you to retrieve logs pertaining to:
+
+- requests to authorization resources to help troubleshoot authentication issues
+- configmap and secret changes in all namespaces at the Metadata audit level
+- changes to resources at RequestResponse level (maximum verbosity) to create, patch, update and delete verbs
+- changes to resources at Request level for other verbs
+- all other requests at the Metadata level
+
+Note:
+
+- No logs for requests by the "system:kube-proxy" on endpoints or services
+- Low level of verbosity (level Metadata) for endpoints containing sensitive data like tokenreview
+- No logs for any request in the RequestReceived stage
+- No logs for health checks and requests for apiserver metrics
+
+For details about information captured in Kubernetes Audit logs you can refer to the [Kubernetes public documentation](https://kubernetes.io/docs/reference/config-api/apiserver-audit.v1/).
+
+Below is an example of an audit log generated by a Kubernetes cluster. *Note the example is not exhaustive.*
+
+```json
+{
+   "kind":"Event",
+   "apiVersion":"audit.k8s.io/v1",
+   "level":"Metadata",
+   "auditID":"XXXXXXX-XXXX-XXXX-XXXX-XXXXXXXX",
+   "stage":"ResponseStarted",
+   "requestURI":"/api/v1/namespaces/searching-worker/configmaps?......",
+   "verb":"watch",
+   "user":{
+      "username":"system:node:search-workers-node-81097c",
+      "groups":[
+         "system:nodes",
+         "system:authenticated"
+      ]
+   },
+   "sourceIPs":[
+      "10.10.0.43"
+   ],
+   "userAgent":"kubelet/v1.26.4 (linux/amd64) kubernetes/f89670c",
+   "objectRef":{
+      "resource":"configmaps",
+      "namespace":"searching-worker",
+      "name":"kube-root-ca.crt",
+      "apiVersion":"v1"
+   },
+   "responseStatus":{
+      "metadata":{
+
+      },
+      "code":200
+   },
+   "requestReceivedTimestamp":"2024-01-01T11:59:06.428015Z",
+   "stageTimestamp":"2024-01-01T11:59:06.428962Z",
+   "annotations":{
+      "authorization.k8s.io/decision":"allow",
+      "authorization.k8s.io/reason":""
+   }
+}
+```
+
+## Instructions
+
+How to enable Audit Log Forwarding
+
+### Enabling Audit Log Forwarding using the OVHcloud Control Panel
+
+#### Create and see existing streams
+
+You can find your streamId, in the Logs Data Platform section of the [OVHcloud control panel](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.co.uk/&ovhSubsidiary=GB).
+
+- Go to the `Data Stream`{.action} tab of your Logs Data Platform account and `Edit`{.action} the target Data Stream.
+
+https://www.ovh.com/manager/#/dedicated/dbaas/logs/{ldp-account-id}/streams/home
```

Comment:
If this URL is meant to be clickable, it should be formatted as such
OR I suggest replacing those with screenshots from the OVHcloud Control Panel.

Reply:
Remplaced by sceenshots (when available) or instructions. We will add more screenshots when EN translation will be available on the OVH Control Panel